src/components/Dropdown.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect, useCallback } from "react";
```

Comment:
A little help does not harm. but maybe remove the import of React from react ;)

Reply:
You're right